frontend/javascripts/oxalis/view/settings/histogram_view.js

```diff
@@ -132,9 +197,41 @@ class Histogram extends React.PureComponent<HistogramProps> {
     }
   };
 
+  tipFormatter = (value: number) =>
+    value > 10000 ? value.toExponential() : roundTo(value, 2).toString();
+
+  // eslint-disable-next-line react/sort-comp
+  updateMinimumDebounced = _.debounce(
+    (value, layerName) => this.props.onChangeLayer(layerName, "min", value),
+    1000,
```

Comment:
```suggestion
    500,
```

I would suggest to half the debounce time to make it feel a little more snappy